app/src/main/res/layout-land/wallets_layout.xml

```diff
@@ -0,0 +1,248 @@
+        />
+  </androidx.constraintlayout.widget.ConstraintLayout>
+  <View
+      android:id="@+id/header_separator"
+      android:layout_width="match_parent"
+      android:layout_height="2dp"
+      android:layout_marginTop="@dimen/bottom_sheet_peek_height"
+      android:background="#f3f3f3"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      />
+  <androidx.core.widget.NestedScrollView
+      android:layout_width="match_parent"
+      android:layout_height="0dp"
+      android:layout_marginTop="1dp"
+      app:layout_constraintBottom_toTopOf="@id/bottom_sheet_bottom_separator"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/header_separator"
+      >
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:clipChildren="false"
+        android:clipToPadding="false"
+        >
+      <TextView
+          android:id="@+id/active_wallet_header"
+          style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="16dp"
+          android:layout_marginTop="16dp"
+          android:letterSpacing="0.01"
+          android:text="@string/wallets_active_wallet_title"
+          android:textColor="@color/black_87_alpha"
+          android:textSize="16sp"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="parent"
+          />
+
+      <include
+          android:id="@+id/active_wallet_card"
+          layout="@layout/active_wallet_card"
+          android:layout_width="match_parent"
+          android:layout_height="72dp"
+          android:layout_marginTop="12dp"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@id/active_wallet_header"
+          />
+
+      <TextView
+          android:id="@+id/other_wallets_header"
+          style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="16dp"
+          android:layout_marginTop="24dp"
+          android:letterSpacing="0.01"
+          android:text="@string/wallets_2nd_view_other_wallets_title"
+          android:textColor="@color/black_87_alpha"
+          android:textSize="16sp"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@id/active_wallet_card"
+          />
+
+      <androidx.recyclerview.widget.RecyclerView
+          android:id="@+id/other_wallets_cards_recycler"
+          android:layout_width="match_parent"
+          android:layout_height="match_parent"
```

Comment:
orientation and layout manager can be defined in XML.
apply that to the portrait file